_transcripts/008-oli-miri.md

```diff
@@ -77,7 +77,7 @@ which is essentially, just like something working on the syntax tree of the
 language, and like, the so-called constant folding. So it folds syntax
 components to get— so if it sees, like, "2 + 2", it knows, okay, there's an
 integer, there's an integer, and there's a plus, I can figure out that this is
-4. It is very simple and is also very limited, because once you get into, like,
+4\. It is very simple and is also very limited, because once you get into, like,
```

Comment:
I'm curious about this fix. I was thinking just lift the 4. to the line above, but if this works...

Reply:
I did it this way was to prevent the problem from returning if I re-word-wrap the file.  Which almost certainly wouldn't happen once the PR is public, but before I wrote scripts to help me automate things I would sometimes accidentally re-introduce formatting errors by repeating the word wrap step.

My current workflow is to do all of my editing without any line wrapping, and then do word-wrapping at the very end, right before sending a PR containing all of my work squashed into a single commit.  Then after that point, let changes accumulate without re-word-wrapping everything, so that git history is pristine.  I'm probably over-thinking it though...